css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,98 @@
+$progress-bar-height-medium: 0.125rem !default;
+$progress-bar-height-large: 0.25rem !default;
+
+$progress-bar-container-background-color: $secondary-background !default;
+$progress-bar-background-color: $primary !default;
+$progress-bar-success-background-color: $success !default;
+$progress-bar-warning-background-color: $warning !default;
+$progress-bar-danger-background-color: $danger !default;
+$progress-bar-border-radius-rounded: $border-radius-rounded !default;
```

Comment:
Nit: rename to `$progress-bar-border-radius`. it's a bit more flexible name for the variable which stands for default value